example/src/main/scala/com/avast/server/toolkit/example/Main.scala

```diff
@@ -0,0 +1,26 @@
+package com.avast.server.toolkit.example
+
+import cats.effect.Resource
+import com.avast.server.toolkit.example.config.Configuration
+import com.avast.server.toolkit.pureconfig.PureConfigModule
+import zio.interop.catz._
+import zio.{Task, ZIO}
+
+object Main extends CatsApp {
+
+  def program: Resource[Task, Unit] = {
+    for {
+      configuration <- Resource.liftF(PureConfigModule.makeOrThrow[Task, Configuration])
```

Comment:
I don't think that raising exception is the the best way to solve invalid configuration. I understand it's convenient, but the stack trace is completely useless.

Reply:
I understand your view but I think this approach is pragmatic. Usually loading of the configuration will be the first thing in your application and if that fails you definitely do not want to continue - it's fatal error. This is why I think throwing an exception is just fine, not because of the stack trace which is as you say useless, but because it stops your application immediately and you do not have to wrap the whole initialization into `EitherT` or find other solution.